### ISSUE TITLE
Check for unused function arguments

### ISSUE THREAD

**Issue by dreamflasher** [wemake-services/wemake-python-styleguide#2326]
# Rule request

## Thesis

Check for unused function arguments. See:
https://pypi.org/project/flake8-unused-arguments/
With the following settings:
* don't show warnings for abstract functions
* don't show warnings for empty functions
* don't show warnings for unused *args and **kwargs

## Reasoning

It's in most cases a forgotten argument, Pylance detects this, similar reasoning as with unused variables.

**Comment by dreamflasher** [comment]
@sobolevn what do you think about this?

**Comment by sobolevn** [comment]
Hi! Thanks for raising this! 

Context: https://github.com/wemake-services/wemake-python-styleguide/issues/317

So, why did I revert this? Because argument names are a part of a function signature.
Consider this example:

```python
def some(x, y):
    print(x)  # `y` is not used

some(x=1, y=2)
```

Now, we change it to be `def some(x, _y)` to follow our rule (or even `def some(x)`). But, with this change `some(x=1, y=2)` won't work! I know that in this example it will be easy to refactor the call, but it is not possible in 100% of cases.

There are:
- APIs that expect specific names, desipe the fact you might not use all arguments
- Inheritance
- Callbacks
- etc

So, I don't recommend using this practice.
